de.tudarmstadt.ukp.wikipedia.parser/pom.xml

```diff
@@ -44,14 +44,14 @@ Please use the SWEBLE parser (http://sweble.org/) for parsing MediaWiki markup.
 		  <artifactId>slf4j-api</artifactId>
 	  </dependency>
 	  <dependency>
-		  <groupId>org.slf4j</groupId>
-		  <artifactId>slf4j-log4j12</artifactId>
-		  <scope>test</scope>
+		  <groupId>org.apache.logging.log4j</groupId>
```

Comment:
Why provided instead of test?

Reply:
Think it is just CnP from my side.

It seems, that `parser` is intended as a library module. Other modules such as "dumper", etc. are more or less CLI modules (with provided log4j2 deps). Will fix it.